docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -257,8 +257,24 @@ the ESPW.
 These commands administer a deployment of the central components ---
 the kcp server, PKI, and the central KubeStellar components --- in a
 Kubernetes cluster that will be referred to as "the hosting cluster".
-These are framed as "kubectl plugins" and thus need to be explicitly
-or implicitly given a kubeconfig file for the hosting cluster.
+These commands are framed as "kubectl plugins" and thus need to be
+explicitly or implicitly given a kubeconfig file for the hosting
```

Comment:
What do you mean "implicitly" ? using env var ?

Reply:
yes